packages/lodestar-cli/src/commands/beacon.ts

```diff
@@ -33,6 +43,11 @@ export class BeaconNodeCommand implements ICliCommand {
       .command("beacon")
       .description("Start lodestar node")
       .option("-c, --configFile [config_file]", "Config file path")
+      .option("-p, --preset [preset]", "Minimal/mainnet", "minimal")
+      .option("-s, --genesisState [params]", "Start chain from known state")
```

Comment:
We will likely not want to be starting from a known genesis state but rather start with an eth1 provider and eth1 deposit contract address and use the eth1 chain and the deposit contract to determine when/what the genesis state is.